docs/docs-content/release-notes.md

```diff
@@ -11,6 +11,18 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## November 7, 2023 - Release 4.1.9
+
+This release contains the following minor bug fixes related to protected Helm Chart and Open Container Initiative (OCI) registries.
+
+- Where Palette displayed the UID of private or protected Helm Chart registries in a drop-down menu during cluster profile creation, Palette now correctly displays the registry name.
```

Comment:
```suggestion
- An issue where Palette was displaying the UID of protected Helm Chart and Open Container Initiative (OCI) registries instead of the expected registry name is now corrected.
```
Rita, could we simplify the sentence to something along these lines maybe?